apps/condo/domains/meter/tasks/deleteReadingsOfDeletedMeter.js

```diff
@@ -1,26 +1,37 @@
 const { getSchemaCtx, find } = require('@open-condo/keystone/schema')
 const { createTask } = require('@open-condo/keystone/tasks')
 
-const { MeterReading } = require('../utils/serverSchema')
+const { MeterReading } = require('@condo/domains/meter/utils/serverSchema')
+const { CommunalMeterReading } = require('@condo/domains/meter/utils/serverSchema')
 
 /**
  * Soft delete meter readings after soft delete meter
  */
-async function deleteReadingsOfDeletedMeter (deletedMeter, deletedMeterAt) {
+async function deleteReadingsOfDeletedMeter (deletedMeter, deletedMeterAt, isCommunalMeter = false) {
```

Comment:
Where's the usage of this task? Need to understand logic when working with contractors, should it be deleted or not...